conceptpower/classes.py

```diff
@@ -157,4 +158,60 @@ def get_type(self, uri):
                 data['supertype_id'] = snode.get('supertype_id')
                 data['supertype_uri'] = snode.get('supertype_uri')
         return data
-        
\ No newline at end of file
+
+    def create(self,user,password,required_concept_data,**kwargs):
+        """
+        Adds a concept to the conceptlist by using rest api
+
+        Parameters
+        -----------
+        user: str
+             userId of the person who wants to add concept
+        password: str
+                 password of the person who wants to add concept
+        required_values: dict
+                        required keys
+                        key: "word"
+                                name of the word
+                        key: "pos"
+                                part of speech of the word
+                        key: "conceptlist"
+                                name of the concept list it belongs to,
+                        key: "description"
+                                description of the word
+                        key: "types"
+                                types of the word in the mentioned order
+
+                        For example:
+                        required_concept_data = {"word":"kitty","pos":"noun","conceptlist":"mylist",
+                                                 "description":"soft kitten","types":"LivingBeing"}
+
+        kwargs: kwargs
+                storing additional attributes passed
+
+        :return boolean: whether the concept has been succesfully added or not
+        """
+
+        #checking whether all required attributes are present in required_concept_data
```

Comment:
Remember: put a space after the hash, and then start with a capital letter. E.g. ```# Checking ...```